### DISCUSSION TITLE
Badges in Karazhan

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#6201] [Suggestions]
So, as i'm am sure a lot of people have noticed some parts in Kara drop badges others don't https://github.com/chromiecraft/chromiecraft/issues/6169 mainly chess event as seen in this link here
But in my opinion and for the betterment of the general playerbase I think that every boss in Kara should drop badges

It is true that kara didn't drop badges until later tbc and while I imagine that is the plan regarding badges I think that it would be much better for the player base if Kara dropped badges and here is why.
I don't have any issue with karazhan tuning as a whole. I think it suits the raid very well as its not supposed to be a very difficult raid and I think the tuning reflects that. That being said if the raid is pretty doable with people 50% heroic gear I think that letting new players or alts do karazhan kill 8/10 which is pretty standard for a pug and get 8 badges would be a good thing. Letting new players who may have troubles getting into heroics because of queue times or just the nature of being in a different time zone are able to get badges and thus get badge gear. Karazhan not dropping badges realistically only affects the newer players, as The players who have already been playing a long time don't need badges as they have plently from stock piling.   They honestly might not even need Kara except for a couple of items to achieve full best in slot.
And since the player base seems to be increasing I truly believe this is the best way to help players gear faster since heroics will not pop or you sit in queue for 2+hours if you dont queue at a very specific time

**Comment by balleny** [comment]
the concept of not dropping badges yet is so that when raids start dropping badges and lots of high-end gear becomes available (ZA and Sunwell) from the vendor, there is incentive to farm those badges (given items then will cost up to 150 badges per piece).

that will help increase the demand for old raids like kara at an advantaged stage of content, help new players find e.g. kara or t4 and t5 groups etc.

moreover, players that just raid-log wont have an advantage of having all the items pre-farmed. If you add badges to raids, there is even less incentive for people to run heroics to get gear.
